test/python/visualization/test_circuit_text_drawer.py

```diff
@@ -2105,15 +2105,15 @@ def test_controlled_composite_gate_bot(self):
     def test_controlled_composite_gate_top_bot(self):
         """Controlled composite gates (top and bottom) """
         expected = '\n'.join(["                ",
-                              "q_0: |0>───■────",
+                              "q_0: |0>───o────",
                               "        ┌──┴───┐",
                               "q_1: |0>┤0     ├",
                               "        │      │",
                               "q_2: |0>┤1 Ghz ├",
                               "        │      │",
                               "q_3: |0>┤2     ├",
                               "        └──┬───┘",
-                              "q_4: |0>───o────",
+                              "q_4: |0>───■────",
```

Comment:
I don't think this change is correct. The gate is applied as 
```python
        ccghz = ghz.control(2, ctrl_state='01')
        circuit.append(ccghz, [4, 0, 1, 2, 3])
```
`ctrl_state='01' ` means open-control on the more significant control line, and closed-control on the less significant control line. The control lines are q4 and q0. q4 is more significant, so should be open.

Reply:
Ok. I was following the example from #3957 
```qr = QuantumRegister(4, 'q')
circuit = QuantumCircuit(qr)
circuit.append(U2Gate(pi, -5 * pi / 8).control(3, ctrl_state='100'),
               [qr[0], qr[3], qr[2], qr[1]])
circuit.draw('text')
                         
q_0: |0>────────o────────
        ┌───────┴───────┐
q_1: |0>┤ U2(pi,-5pi/8) ├
        └───────┬───────┘
q_2: |0>────────■────────
                │        
q_3: |0>────────o────────
```
and the code in each of the 3 drawers,

`        ctrl_state = "{0:b}".format(instruction.op.ctrl_state).rjust(num_ctrl_qubits, '0')[::-1]
`

which reverses ctrl_state and then applies it to each of the control qubits left to right. Is this different for the multigates?
And while we're discussing, the cz in text.py appears as an open on top and closed on the bottom, whereas in mpl it is both closed. Which way should this be?
Thanks.

Reply:
the output should be the same for all 3 drawers. please raise an issue if it is not. I'm not sure what they do internally, but i can say that the output that was there is correct for text drawer.

> cz in text.py appears as an open on top and closed on the bottom, whereas in mpl it is both closed

CZ with what ctrl_state? one of the controls should be open if there's a '0' in the ctrl_state.

Reply:
- Ok on the ghz gate as it was.
- On the cz, I was asking what the default was, but I see that cx, cy, & cz all default to ctrl_state 1, so it works fine in text.py. matplotlib.py does not address the ctrl_state for cz and needs to be fixed.
- See my comment in #4180. If we do what I suggested, I'll fix the matplotlib.py with this as well.